packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -60,75 +62,119 @@ class _AccountPictures extends StatelessWidget {
   }
 }
 
-class _AccountDetails extends StatelessWidget {
+class _AccountDetails extends StatefulWidget {
   const _AccountDetails({
-    Key key,
     @required this.accountName,
     @required this.accountEmail,
     this.onTap,
     this.isOpen,
-  }) : super(key: key);
+  });
 
   final Widget accountName;
   final Widget accountEmail;
   final VoidCallback onTap;
   final bool isOpen;
 
+  @override
+  _AccountDetailsState createState() => new _AccountDetailsState();
+}
+
+class _AccountDetailsState extends State<_AccountDetails> with SingleTickerProviderStateMixin {
+  Animation<double> _animation;
+  AnimationController _controller;
+
+  @override
+  void initState () {
+    super.initState();
+
+    _controller = new AnimationController(
+      value: widget.isOpen ? 1.0 : 0.0,
+      duration: const Duration(milliseconds: 200),
+      vsync: this,
+    );
+    _animation = new CurvedAnimation(
+        parent: _controller,
+        curve: Curves.easeInOut,
```

Comment:
why this curve?

Reply:
I picked `easeInOut` because that's what [the Material guidelines recommend](https://material.io/guidelines/motion/duration-easing.html#duration-easing-natural-easing-curves). Should this be a different curve?

Reply:
Ah, I see. That's actually `Curves.fastOutSlowIn` (the name matches Android's name for the same).